packages/app/src/app/pages/common/Modals/SignInForTemplates/SignInForTemplates.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { Button } from '@codesandbox/common/lib/components/Button';
+import { useSignals } from 'app/store';
+import { Container } from '../LiveSessionEnded/elements';
+import SignInButton from '../../SignInButton';
+import { Heading, Explanation } from '../elements';
+import { Close, Buttons } from './elements';
+
+export const SignInForTemplates = () => {
+  const { modalClosed } = useSignals();
+  return (
+    <Container>
+      <Close onClick={() => modalClosed()} />
```

Comment:
@SaraVieira This can just be

```suggestion
      <Close onClick={modalClosed} />
```

Reply:
I wouldn't pass events to Cerebral, calling this function without arguments is cleaner IMO.